app/src/main/java/com/blogspot/e_kanivets/moneytracker/controller/external/Head.java

```diff
@@ -10,8 +10,9 @@ public interface Head {
     String TIME = "time";
     String TITLE = "title";
     String CATEGORY = "category";
+    String NOTES = "notes";
     String PRICE = "price";
     String CURRENCY = "currency";
     String DELIMITER = ";";
-    int COLUMN_COUNT = 5;
+    int COLUMN_COUNT = 7;
```

Comment:
6?

Reply:
Hey I've fixed most of the other things. But here we will also be adding ACCOUNT_ID to export right. so after that COLUMN_COUNT should become 7 right. So should I make it six and take care of it during the merge or something else.